apps/connect/src/hooks/useDocumentDriveServer.ts

```diff
@@ -652,7 +652,7 @@ export function useDocumentDriveServer() {
     );
 
     const onSyncStatus = useCallback(
-        (cb?: (driveId: string, status: SyncStatus, error?: Error) => void) => {
+        (cb: (driveId: string, status: SyncStatus, error?: Error) => void) => {
```

Comment:
document drive expects the callback to be defined. if an empty one needs to be used, pass an empty function